### ISSUE TITLE
why is there no reproducibility of results?

### ISSUE THREAD

**Issue by Ludmila6114** [Joker-Jerome/UTMOST#13]
So, i tried single_tissue test for my summarystat gwas file and got the tissues. Then i created new directory for output and also perform this test with the same input gwas file. But i have different results! Firstly, i got only one tissue compare to 4 tissues later. Why? 
I use your tissue_list, your sample_data/weight_db and covariance. The only 

for tissue in ${TISSUE_GTEx[@]}; do /usr/bin/python2.7 ./single_tissue_association_test.py --model_db_path sample_data/weight_db_GTEx/${tissue}.db --covariance sample_data/covariance_tissue/${tissue}.txt.gz --gwas_folder sample_data/GWAS --gwas_file_pattern AN_Utmost.txt --snp_column SNP --effect_allele_column A1 --non_effect_allele_column A2 --beta_column b --pvalue_column p --output_file sample_data/AN_res/${tissue}.csv; done

for tissue in ${TISSUE_GTEx[@]}; do /usr/bin/python2.7 ./single_tissue_association_test.py --model_db_path sample_data/weight_db_GTEx/${tissue}.db --covariance sample_data/covariance_tissue/${tissue}.txt.gz --gwas_folder sample_data/GWAS --gwas_file_pattern AN_Utmost.txt --snp_column SNP --effect_allele_column A1 --non_effect_allele_column A2 --beta_column b --pvalue_column p --output_file sample_data/AN_NEW_res/${tissue}.csv; done

i have difference between AN_res and AN_NEW_res. Why?

**Comment by Ludmila6114** [comment]
The same is with your test GWAS file... 
SNP_gwas_mc_merge_nogc.tbl.uniq

**Comment by Joker-Jerome** [comment]
It is pretty weird that you got results from different tissues. Were those single tissue test programs finished?

**Comment by Ludmila6114** [comment]
yes, sure!

**Comment by Ludmila6114** [comment]
it seems like the problem is with local memory, i'll check!

**Comment by yiminghu** [comment]
@Ludmila6114 was this resolved?

**Comment by Ludmila6114** [comment]
@yiminghu Yes, it was just a memory problem, sorry for bother you before checking it myself.
